Resources/views/default.html.twig

```diff
@@ -0,0 +1,56 @@
+{% apply spaceless %}
```

Comment:
I actually started out this way, but I was ending up with double rendered markup and flipped to a full template.  Which, as I started building, I kind of liked better than the block based approach (and this isn't much different than [Laravel's Pagination component](https://github.com/illuminate/pagination) as far as how templating was done there.

And by the time I started looking at how [the FormExtension](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Twig/Extension/FormExtension.php) actually interacts with Twig, it started coming across as being a highly technically complex integration to be able to "simplify" rendering down to a handful of reusable blocks.

Reply:
I suppose the Laravel's pagination component is also closely modeled after PagerFanta's inner logic.
What do you mean by "double rendered markup"? Sounds to me like a silly bug which manages to evade your eyes late at night, and then pops right into the center of your attention the next morning.
I still stand by my idea here, assuming it can be implemented without too much hassle or technical complexity. I can try to do this myself and make a PR into your branch if I succeed.

Reply:
If you've got any ideas then I'll gladly take a look.

I originally started with a template with nothing but blocks, and rendering that caused the double render.

```twig
{% block container %}
    <nav></nav>
{% endblock %}
```

The output when I was running the integration tests was `<nav></nav><nav></nav>`, no idea why.

Reply:
> If you've got any ideas then I'll gladly take a look.
> 
> I originally started with a template with nothing but blocks, and rendering that caused the double render.
> 
> ```
> {% block container %}
>     <nav></nav>
> {% endblock %}
> ```
> 
> The output when I was running the integration tests was `<nav></nav><nav></nav>`, no idea why.

Because you called `$this->twig->render()` instead of constructing a TemplateWrapper object and calling `renderBlock()` on it, I guess?